cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/KafkaCruiseControlServlet.java

```diff
@@ -671,6 +680,26 @@ private boolean getPartitionLoad(HttpServletRequest request, HttpServletResponse
       String endMsString = request.getParameter(END_MS_PARAM);
       startMs = startMsString == null ? -1L : Long.parseLong(startMsString);
       endMs = endMsString == null ? System.currentTimeMillis() : Long.parseLong(endMsString);
+      String entriesString = request.getParameter(ENTRIES_PARAM);
+      entries = entriesString == null ? Integer.MAX_VALUE : Integer.parseInt(entriesString);
+      String topicString = request.getParameter(TOPIC_PARAM);
+      if (topicString != null) {
+        topic = Pattern.compile(topicString);
+      } else {
+        topic = null;
+      }
+      String partitionString = request.getParameter(PARTITION_PARAM);
+      if (partitionString == null) {
+        partitionLowerBoundary = Integer.MIN_VALUE;
+        partitionUpperBoundary = Integer.MAX_VALUE;
+      } else if (!partitionString.contains("-")) {
+        partitionLowerBoundary = Integer.parseInt(partitionString);
+        partitionUpperBoundary = partitionLowerBoundary;
+      } else {
```

Comment:
Sorry for not adding this in the previous patch but we can also we can add a comma-separated list of partitions too.